### ISSUE TITLE
Exceptions when building the project

### ISSUE THREAD

**Issue by bemxio** [ennuo/toolkit#29]
When I tried to build the project cloned from the v0.1.1 tag by running the `mvn package` command, I got a bunch of errors:
```
[INFO] Scanning for projects...
[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[ERROR] 'dependencies.dependency.version' for org.lwjgl:lwjgl:jar:${lwjgl.natives} is missing. @ line 154, column 21
[ERROR] 'dependencies.dependency.version' for org.lwjgl:lwjgl-glfw:jar:${lwjgl.natives} is missing. @ line 159, column 21
[ERROR] 'dependencies.dependency.version' for org.lwjgl:lwjgl-opengl:jar:${lwjgl.natives} is missing. @ line 164, column 21
[ERROR] 'dependencies.dependency.version' for org.lwjgl:lwjgl:jar:${lwjgl.natives} is missing. @ line 163, column 21
[ERROR] 'dependencies.dependency.version' for org.lwjgl:lwjgl-tinyfd:jar:${lwjgl.natives} is missing. @ line 168, column 21
 @ 
[ERROR] The build could not read 2 projects -> [Help 1]
[ERROR]   
[ERROR]   The project ennuo:cwlib-gl:0.1 (/data/data/com.termux/files/home/toolkit/lib/cwlib-gl/pom.xml) has 3 errors
[ERROR]     'dependencies.dependency.version' for org.lwjgl:lwjgl:jar:${lwjgl.natives} is missing. @ line 154, column 21
[ERROR]     'dependencies.dependency.version' for org.lwjgl:lwjgl-glfw:jar:${lwjgl.natives} is missing. @ line 159, column 21
[ERROR]     'dependencies.dependency.version' for org.lwjgl:lwjgl-opengl:jar:${lwjgl.natives} is missing. @ line 164, column 21
[ERROR]   
[ERROR]   The project ennuo:toolkit:0.1 (/data/data/com.termux/files/home/toolkit/toolkit/pom.xml) has 2 errors
[ERROR]     'dependencies.dependency.version' for org.lwjgl:lwjgl:jar:${lwjgl.natives} is missing. @ line 163, column 21
[ERROR]     'dependencies.dependency.version' for org.lwjgl:lwjgl-tinyfd:jar:${lwjgl.natives} is missing. @ line 168, column 21
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/ProjectBuildingException
```

I am using Termux (sadly cannot use my PC at the moment, so I need to compile the project on it) with OpenJDK 17 and Maven.

Should I use a different command or perhaps do some other stuff before running `mvn package`?

**Comment by ennuo** [comment]
This project uses LWJGL native dependencies, so you have to specify the profile with the package command.

Valid profiles should be:
- lwjgl-natives-linux-amd64
- lwjgl-natives-macos-aarch64
- lwjgl-natives-macos-x86_64
- lwjgl-natives-windows-amd64

**Comment by bemxio** [comment]
Great timing, was just about to say that after a bit of tinkering, I found out the profile for Linux on 64-bit ARM is missing.

I've decided to try creating a profile for that target and it builds without any issues:
```xml
        <profile>
            <id>lwjgl-natives-linux-aarch64</id>
            <activation>
                <os>
                    <family>unix</family>
                    <arch>aarch64</arch>
                </os>
            </activation>
            <properties>
                <lwjgl.natives>natives-linux-arm64</lwjgl.natives>
            </properties>
        </profile>
```

Also tried to run it and it seems to be all good, although I haven't tried opening any files yet.

It would be nice to add that profile and the 32-bit equivalent, not only for Termux masochists but also for those that want to use a Raspberry Pi.
If I don't come across any issues with the project built using that profile, I can make a PR adding those profiles to the appropriate POMs.
